### PR TITLE
Allow users to easily modify the date/time format used in events

### LINKED ISSUE #158: Make the date and time output more configurable
When displaying and inputting dates and times make the format easily customizable via a config file.  This would be useful for the event datetime picker and displays.

### PR DESCRIPTION
**Pull request by mrrsm**
Defaults to ISO 8601 date format and allows the user to configure the date/time formats used for events.
This fixes #158

### CODE REVIEW

File: config/date.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    | PHP Format: NOTE this must be compatiable with the javascript date
+    |--------------------------------------------------------------------------
+    |
+    | Take a look at http://php.net/manual/en/function.date.php
+    |
+    | The default is ISO-8601
+    |
+    */
+
+    'php_format' => 'Y-m-d H:i',
```
Comment (GrahamCampbell) [inline config/date.php]: Can we keep the defaults the same as they currently are please.

Comment (mrrsm) [inline config/date.php]: I feel like using an ISO standard as the default makes more sense than picking a specific region's date/time format.  It should cause less confusion for someone starting to use the project.  Is there a strong reason to use the current default?

<sub>sidenote: The relevant [XKCD](https://xkcd.com/1179/)</sub>

Comment (GrahamCampbell) [inline config/date.php]: I disagree. I like UK readable dates by default.

**Comment by GrahamCampbell** [comment]
Sorry this took me so long to get to. Thank you very much for the contribution.

**Pull request merged**